Client/Frontend/Home/ASHeaderView.swift

```diff
@@ -17,7 +17,7 @@ class ASHeaderView: UICollectionReusableView {
         let titleLabel = UILabel()
         titleLabel.text = self.title
         titleLabel.textColor = UIColor.theme.homePanel.activityStreamHeaderText
-        titleLabel.font = UIFont.preferredFont(forTextStyle: .title3)
+        titleLabel.font = UIFont.systemFont(ofSize: 20, weight: .bold)
```

Comment:
Is it possible to use a variable here that's more descriptive than a magic number? That way, if/when we update this font size, we can just do it easily.